Gems/Atom/RPI/Code/Source/RPI.Edit/Shader/ShaderVariantListSourceData.cpp

```diff
@@ -29,9 +29,10 @@ namespace AZ
                     ;
 
                 serializeContext->Class<ShaderVariantListSourceData>()
-                    ->Version(1)
+                    ->Version(2)
```

Comment:
Let's add a quick comment to induce the changes going with this version. We do this in a bunch of other places too.